[2021-03-04 20:04:24,148] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: clean_data.load_data 2021-03-04T20:04:18.948253+00:00 [queued]>
[2021-03-04 20:04:24,170] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: clean_data.load_data 2021-03-04T20:04:18.948253+00:00 [queued]>
[2021-03-04 20:04:24,170] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-03-04 20:04:24,171] {taskinstance.py:881} INFO - Starting attempt 1 of 2
[2021-03-04 20:04:24,171] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-03-04 20:04:24,217] {taskinstance.py:901} INFO - Executing <Task(PythonOperator): load_data> on 2021-03-04T20:04:18.948253+00:00
[2021-03-04 20:04:24,224] {standard_task_runner.py:54} INFO - Started process 4163 to run task
[2021-03-04 20:04:24,268] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'clean_data', 'load_data', '2021-03-04T20:04:18.948253+00:00', '--job_id', '27', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/data_clean.py', '--cfg_path', '/tmp/tmp90c6w2zt']
[2021-03-04 20:04:24,274] {standard_task_runner.py:78} INFO - Job 27: Subtask load_data
[2021-03-04 20:04:24,346] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: clean_data.load_data 2021-03-04T20:04:18.948253+00:00 [running]> 6623e80cd770
[2021-03-04 20:04:24,411] {logging_mixin.py:112} WARNING - /home/airflow/.local/lib/python3.6/site-packages/pandas/core/generic.py:2612: UserWarning: The spaces in these column names will not be changed. In pandas versions < 0.14, spaces were converted to underscores.
  method=method,
[2021-03-04 20:04:24,787] {python_operator.py:114} INFO - Done. Returned value was: None
[2021-03-04 20:04:24,798] {taskinstance.py:1070} INFO - Marking task as SUCCESS.dag_id=clean_data, task_id=load_data, execution_date=20210304T200418, start_date=20210304T200424, end_date=20210304T200424
[2021-03-04 20:04:29,133] {local_task_job.py:102} INFO - Task exited with return code 0
